wave_utils/src/world_frame_conversions.cpp

```diff
@@ -0,0 +1,206 @@
+/* Copyright (c) 2017, Waterloo Autonomous Vehicles Laboratory (WAVELab),
+ * Waterloo Intelligent Systems Engineering Lab (WISELab),
+ * University of Waterloo.
+ *
+ * Refer to the accompanying LICENSE file for license information.
+ *
+ * ############################################################################
+ ******************************************************************************
+ |                                                                            |
+ |                         /\/\__/\_/\      /\_/\__/\/\                       |
+ |                         \          \____/          /                       |
+ |                          '----________________----'                        |
+ |                              /                \                            |
+ |                            O/_____/_______/____\O                          |
+ |                            /____________________\                          |
+ |                           /    (#UNIVERSITY#)    \                         |
+ |                           |[**](#OFWATERLOO#)[**]|                         |
+ |                           \______________________/                         |
+ |                            |_""__|_,----,_|__""_|                          |
+ |                            ! !                ! !                          |
+ |                            '-'                '-'                          |
+ |       __    _   _  _____  ___  __  _  ___  _    _  ___  ___   ____  ____   |
+ |      /  \  | | | ||_   _|/ _ \|  \| |/ _ \| \  / |/ _ \/ _ \ /     |       |
+ |     / /\ \ | |_| |  | |  ||_||| |\  |||_|||  \/  |||_||||_|| \===\ |====   |
+ |    /_/  \_\|_____|  |_|  \___/|_| \_|\___/|_|\/|_|\___/\___/ ____/ |____   |
+ |                                                                            |
+ ******************************************************************************
+ * ############################################################################
+ *
+ * File: world_frame_conversions.cpp
+ * Desc: Implementation file for world frame conversion functions
+ * Auth: Michael Smart <michael.smart@uwaterloo.ca>
+ *
+ * ############################################################################
+*/
+
+#include <Eigen/Core>
+#include "wave/utils/world_frame_conversions.hpp"
+
+namespace wave {
+
+void ecefPointFromLLH(const double llh[3], double ecef[3]) {
```

Comment:
just reiterating my point from above regarding the use of raw arrays. That's my only concern with this file.